### PR TITLE
Integrate 8/25 Nightly RN Build

### PR DESCRIPTION
**Pull request by NickGerleman**
Commits: https://github.com/facebook/react-native/compare/12543d557...9b973f32e

This is the last build before the RNTester redesign. No non-mechanical changes needed that I could find.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/6288)

### CODE REVIEW

File: packages/@react-native/repo-config/overrides.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
-  "baseVersion": "0.0.0-12543d557",
+  "baseVersion": "0.0.0-9b973f32e",
   "overrides": [
     {
-      "type": "derived",
+      "type": "copy",
```
Comment (NickGerleman) [inline packages/@react-native/repo-config/overrides.json:5]: Previously patched out a line pulling in a file-version of eslint config that is now more permissive.

File: packages/@react-native-windows/tester/package.json [hunk 2]
```diff
@@ -11,8 +11,10 @@
     "validate-overrides": "react-native-platform-override validate"
   },
   "dependencies": {
-    "@react-native/tester": "0.0.1",
-    "react-native": "0.0.0-12543d557",
+    "@react-native/tester": "0.0.1"
+  },
+  "peerDependencies": {
```
Comment (NickGerleman) [inline packages/@react-native-windows/tester/package.json:16]: These should have been peerDependencies all along. Core made the same mistake and fixed in one of the commits from this time period, and we do the same.

File: packages/microsoft-reactnative-sampleapps/package.json [hunk 3]
```diff
@@ -26,6 +26,6 @@
     "metro-react-native-babel-preset": "^0.56.0",
     "prettier": "1.19.1",
     "react-native-windows-codegen": "0.1.7",
-    "react-test-renderer": "16.9.0"
+    "react-test-renderer": "16.13.1"
```
Comment (NickGerleman) [inline packages/microsoft-reactnative-sampleapps/package.json:29]: This was bumped by the more liberal package syncing logic we now have. I don't think this or playground actually use the test renderer, but we need it anyway so having the dep of the now correct version should be fine.

File: packages/react-native-win32/.flowconfig [hunk 4]
```diff
@@ -29,9 +30,11 @@
 ; Ignore react-native files in node_modules since they are copied into project root
 .*/node_modules/react-native/.*
 
-; Ignore react-native-windows' target folder. Flow is not compiled with long path support and after running unittests these folders have long paths
+; Ignore react-native-windows' build output. Flow is not compiled with long path support and after running unittests these folders have long paths
```
Comment (NickGerleman) [inline packages/react-native-win32/.flowconfig:33]: At some point we should look at restricting flow a bit so it doesn't do more work than it needs to

File: vnext/overrides.json [hunk 5]
```diff
@@ -252,10 +252,10 @@
       "baseHash": "e982deb23d7ad5312ab0c09508a6d58c152b5be7"
     },
     {
-      "type": "derived",
+      "type": "copy",
```
Comment (NickGerleman) [inline vnext/overrides.json:255]: Core has a builtin stub we were also using in react-native-win32, but we had replaced it with UnimplementedView for some reason. Changed to just use the builtin stub.

**Review by dannyvv — APPROVED** [review]

**Comment by NickGerleman** [comment]
@msftbot merge in 4 hours

**Review by rectified95 — APPROVED** [review]

**Pull request merged**